api/kernel/cpuid.hpp

```diff
@@ -21,7 +21,7 @@
 
 namespace CPUID
 {
-  enum class Feature
```

Comment:
Aha, yeah you have to` static_cast<int>` it if it's an enum class. (or `static_cast<T>` if it's underlying type is explicitly set to `T`, e.g. `enum class Foo : uint8_t {};` )